sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -157,7 +157,13 @@ private[sql] class SharedState(val sparkContext: SparkContext) extends Logging {
 
 object SharedState extends Logging {
   try {
-    URL.setURLStreamHandlerFactory(new FsUrlStreamHandlerFactory())
+    SparkSession.getActiveSession match {
```

Comment:
Ur, is it the same with branch-3.0?

Reply:
Is this because we don't have a configuration patch in branch-2.4?

Reply:
Yes, we dont have the configuration patch